realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -39,6 +39,37 @@
  */
 public class PermissionManager implements Closeable {
 
+    // Reference counted cache equivalent to how Realm instances work.
```

Comment:
Will this always be true? If yes, how do we ensure changes one place are reflected the other place?